test/test_external.py

```diff
@@ -242,5 +243,64 @@ def test_interface_load_cache_examples(tmp_path):
         )
 
 
+def test_cols_to_rows():
+    assert cols_to_rows({"a": [1, 2, "NaN"], "b": [1, "NaN", 3]}) == (
```

Comment:
Just a small question, if there's a cell left empty, how is it handled? Do you impute "NaN" directly? (how is it sent to inference?)

Reply:
Right now it would be left empty but that doesn't work for the inference API so I'll replace with "NaN" instead! Thank you